druid/src/text/movement.rs

```diff
@@ -37,44 +43,98 @@ pub enum Movement {
     EndOfDocument,
 }
 
-/// Compute the result of movement on a selection .
-pub fn movement(m: Movement, s: Selection, text: &impl EditableText, modify: bool) -> Selection {
-    let offset = match m {
+/// Compute the result of movement on a selection.
+///
+/// returns a new selection representing the state after the movement.
+///
+/// If `modify` is true, only the leading edge (the 'end') of the selection
```

Comment:
Is "leading" the right word here? It seems to me "trailing".

Reply:
fair, I guess "active" is the better word.